project/src/main/kotlin/org/ionproject/core/calendar/icalendar/types/UTCOffset.kt

```diff
@@ -0,0 +1,17 @@
+package org.ionproject.core.calendar.icalendar.types
+
+class UTCOffset(
+    private val adding: Boolean,
+    private val hours: Time.Hour,
+    private val minutes: Time.Minute,
+    private val seconds: Time.Second? = null
```

Comment:
That would be something to take into consideration if we were receiving calendar info. Since we are just sending it is better to default to null and not send at all to save on response size and time.